Library/Formula/bar.rb

```diff
@@ -1,11 +1,16 @@
-require 'formula'
-
 class Bar < Formula
-  homepage 'http://www.theiling.de/projects/bar.html'
-  url 'http://www.theiling.de/downloads/bar-1.4-src.tar.bz2'
-  sha1 '00b55ca1d52ca6ed099937692697c2257833c73c'
+  homepage "http://www.theiling.de/projects/bar.html"
+  url "http://www.theiling.de/downloads/bar-1.4-src.tar.bz2"
+  sha1 "00b55ca1d52ca6ed099937692697c2257833c73c"
 
   def install
-    bin.install 'bar'
+    bin.install "bar"
+  end
+
+  test do
+    (testpath/"test1").write "pumpkin"
+    (testpath/"test2").write "latte"
+    system bin/"bar test1 test2 > test3"
+    assert File.read("test3").include?("latte")
```

Comment:
Thanks, I thought that too, but the language threw me off a bit. I'll check it out shortly.